src/main/java/org/edu/model/Station.java

```diff
@@ -5,9 +5,7 @@
 
 
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
+import java.util.*;
```

Comment:
The style guide tells "Wildcard imports, static or otherwise, are not used." ([3.3.1](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)). Please fix it.